app/src/components/misc/SupportPanel/index.tsx

```diff
@@ -0,0 +1,192 @@
+            });
+          }}
+        >
+          <span className="support-panel-label-style">
+            Ask AI<i style={{ color: "#bbbbbb" }}> 24x7 Support</i>
+          </span>
+        </div>
+      ),
+      icon: <AILogo />,
+    },
+    {
+      key: "4",
+      label: (
+        <div
+          className={`support-panel-item-style ${!paidUser && "support-panel-item-disabled"}`}
+          onClick={() => {
+            if (paidUser) {
+              window.$crisp.push(["do", "chat:open"]);
+              trackEvent("support_option_clicked", {
+                type: "chat_with_us",
+              });
+            } else {
+              dispatch(
+                // @ts-ignore
+                actions.toggleActiveModal({
+                  modalName: "pricingModal",
+                  newValue: true,
+                  newProps: { selectedPlan: null, source: "support_option_clicked" },
+                })
+              );
+              trackEvent("support_option_clicked", {
+                type: "upgrade_to_chat_with_us",
+              });
+            }
+            paidUser
+              ? window.$crisp.push(["do", "chat:open"])
+              : dispatch(
+                  // @ts-ignore
+                  actions.toggleActiveModal({
+                    modalName: "pricingModal",
+                    newValue: true,
+                    newProps: { selectedPlan: null, source: "support_option_clicked" },
+                  })
+                );
+          }}
+        >
+          <span className="support-panel-label-style ">
+            Chat With Us
+            {!paidUser && <span className="support-panel-upgrade-badge-style">Upgrade</span>}
+          </span>
+        </div>
+      ),
+      icon: <MessageIcon />,
+    },
+    !slackConnect &&
+      !slackInviteVisibilityStatus &&
+      user.loggedIn &&
+      !paidUser && {
+        key: "5",
+        label: (
+          <a
+            href="mailto:contact@requestly.io"
+            target="_blank"
+            rel="noreferrer"
+            className="support-panel-item-style"
+            onClick={() => {
+              setVisible(false);
+              trackEvent("support_option_clicked", {
+                type: "send_message",
+              });
+            }}
+          >
+            <span className="support-panel-label-style ">Send Message</span>
+          </a>
+        ),
+        icon: <SendMessage />,
+      },
+  ];
+  return (
+    <div className="support-panel-container">
+      <Dropdown menu={{ items: supportItems }} trigger={["click"]} onOpenChange={(state) => setVisible(state)}>
+        <Badge dot={!isSupportChatOpened} size="small">
+          <Button
+            type="primary"
+            size="large"
+            shape="circle"
+            icon={
+              visible || isRequestBotVisible ? <CloseOutlined /> : <ChatIcon className="support-panel-icon-style " />
+            }
+            style={{ fontSize: "1rem" }}
```

Comment:
we also try to avoid inline styles as much as possible. **unless absolutely required, make a separate css file**

And remember to refer to old comment about how css file should be structured